packages/charts/src/components/LineChart/magma-charts.ts

```diff
@@ -69,7 +71,7 @@ const centeredLabelStyles = { textAnchor: 'middle', ...baseLabelStyles };
 const strokeLinecap = 'round';
 const strokeLinejoin = 'round';
 
-export const magmaTheme: any = {
+export const magmaTheme: ThemeInterface = {
```

Comment:
I wonder if this should stay as `any`. This object doesn't match our ThemeInterface (and this line-chart is going to be deprecated soon anyway, so lets skip this one)